### PR TITLE
add change log for the 3.11.0 release

### PR DESCRIPTION
**Pull request by StefanBogdan**

### CODE REVIEW

File: docs/changelog.rst [hunk 1]
```diff
@@ -1,6 +1,65 @@
...
+        .. code-block:: python
+
+            from weaviate import WeaviateErrorRetryConf
+
+            with client.batch(
+                weaviate_error_retries=WeaviateErrorRetryConf(number_retries=3, errors_to_exclude=["Ignore me", "other error to ignore"]),
+            ) as batch:
+                # Your code
+
+    - Exclude errors:
```
Comment (dirkkul) [inline docs/changelog.rst:49]: include

File: docs/changelog.rst [hunk 2]
```diff
@@ -1,6 +1,65 @@
...
+        .. code-block:: python
+
+            from weaviate import WeaviateErrorRetryConf
+
+            with client.batch(
+                weaviate_error_retries=WeaviateErrorRetryConf(number_retries=3),
+            ) as batch:
+                # Your code
+
+    - Exclude errors:
```
Comment (dirkkul) [inline docs/changelog.rst:38]: (all other errors will be retried)

File: docs/changelog.rst (hunk 1 above)
Comment (dirkkul) [reply docs/changelog.rst:49]: And also add that all other errors will be ignored

File: docs/changelog.rst [hunk 3]
```diff
@@ -1,6 +1,65 @@
...
+        .. code-block:: python
+
+            from weaviate import WeaviateErrorRetryConf
+
+            with client.batch(
+                weaviate_error_retries=WeaviateErrorRetryConf(number_retries=3, errors_to_include=["error to retry", "other error to test again"]),
+            ) as batch:
+                # Your code
+
+- Adds new arguments ``sort`` and ``order`` for :meth:`~weaviate.data.DataObject.get`.
```
Comment (dirkkul) [inline docs/changelog.rst:60]: and offset

Comment (StefanBogdan) [reply docs/changelog.rst:60]: Ooo, true. There is no `order`.

**Pull request merged**